### ISSUE TITLE
NullPointer

### ISSUE THREAD

**Issue by poke222444** [CompSci-361/Sprint1#79]
I can start it, but get null pointer every time I try to run it? You should really organize the code as it really looks like a mess, I was trying to contribute but couldn't understand a line of it!

Is this a project for school? What school?

**Comment by AzureKitsune** [comment]
Hello,

We didn't get much time to organize the code as much as we should. This was simply our group project for a CompSci class.

I may eventually go through the code again and clean it up a bit though. 

cc @cjscholl @AdDuAnMn If either of you want to give it a go, feel free.

**Comment by cjscholl** [comment]
Hello,

This is a finished project, which received an A grade by their standards for the course! While I understand that our code could use some additional cleanup (the end of the semester gets really busy with finals and a full course load), I think that there is structure and organization to it (separated into packages, classes, abstract classes/methods, commented code, documentation, test plans and so on).  

What were you trying to contribute and what lines of code are you having trouble understanding? We could clear up confusion surrounding the organization of the code and improve upon it if we knew the specifics. 

Additionally, there are important instructions for starting the program & also using the program listed here in our README documentation: https://github.com/CompSci-361/ChronoTimer. If something is unclear, please open another issue and be as specific as possible. As far as the null pointer is concerned, could you list the process you used for starting the program and running it, along with any error messages and if possible, where it occurred in the code? We need more detail before we can troubleshoot the issue. 

Since there isn't much to work with at this point in time, I'm going to close this issue. Once you provide us with more detail, it can be reopened.

Thanks for your feedback,
Catelyn Scholl
